.github/workflows/build_and_run_selected_test.yml

```diff
@@ -0,0 +1,133 @@
+name: Build and Run Selected Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      test-selection:
+        description: 'Specification of selected test for pytest (e.g. regress/test_setup.py)'
+        required: true
+        type: string
+      run-count:
+        description: 'Number of test runs to perform'
+        required: true
+        type: number
+  pull_request:
+
+defaults:
+  run:
+    shell: bash -euxo pipefail {0}inputs
+
+env:
+  RUST_BACKTRACE: 1
+  COPT: '-Werror'
+
+jobs:
+  tag:
```

Comment:
Now you can use `.github/workflows/_meta.yml` workflow for that (see an example in `.github/workflows/build_and_test.yml`)

Reply:
Thanks, done this way.

Reply:
Yes please, could you please update the workflow to work with workflow_dispatch event

Reply:
The PR for that is ready: https://github.com/neondatabase/neon/pull/11133

Reply:
Fixed.